x-pack/plugins/lens/public/state_management/time_range_middleware.test.ts

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// /*
+//  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+//  * or more contributor license agreements. Licensed under the Elastic License
+//  * 2.0; you may not use this file except in compliance with the Elastic License
+//  * 2.0.
+//  */
+
+import { timeRangeMiddleware } from './time_range_middleware';
+
+import { Observable, Subject } from 'rxjs';
+import { DataPublicPluginStart, esFilters } from '../../../../../src/plugins/data/public';
+import moment from 'moment';
+import { initialState } from './app_slice';
+import { LensAppState } from './types';
+import { PayloadAction } from '@reduxjs/toolkit';
+import { Document } from '../persistence';
+
+const sessionIdSubject = new Subject<string>();
+
+function createMockSearchService() {
```

Comment:
This seems very close to the `mounter.test` mock set of utilities. Can they be shared?